Tests/AlicerceTests/Network/JSONTests.swift

```diff
@@ -1531,6 +1531,155 @@ class JSONTests: XCTestCase {
         }
     }
 
+    func testParseDateAttribute_WithValidIntegerInferredFromFormatter_ShouldReturnAValidDate() {
+
+        let testDate = Date()
+
+        let json = [
+            "testDate" : testDate.timeIntervalSince1970
+        ]
+
+        do {
+            let date = try JSON.parseDateAttribute("testDate",
+                                                   json: json,
+                                                   formatter: Date.init(timeIntervalSince1970:))
+
+            XCTAssertEqual(date.timeIntervalSince1970, testDate.timeIntervalSince1970)
```

Comment:
I know it should be equivalent, but shouldn't we compare `date` and `testDate` directly? 💅

Reply:
yeeaaahhhhh it makes sense 🤔 ok